lib/mandriller/version.rb

```diff
@@ -1,3 +1,3 @@
 module Mandriller
-  VERSION = '0.2.0'
+  VERSION = '0.2.1'
```

Comment:
Definitely. You can set it to whatever you want before merging to master. But since i made some changes i wanted to increased, at least temporarly on the branch.